index.bs

```diff
@@ -549,6 +549,18 @@ interface WebTransport {
   Promise&lt;WebTransportSendStream&gt; createUnidirectionalStream();
   /* a ReadableStream of WebTransportReceiveStream objects */
   readonly attribute ReadableStream incomingUnidirectionalStreams;
+
+  /* if unset, default is 1 second */
+  attribute unsigned long? rateControlFeedbackMinIntervalMs;
+  /* fires when the value changes, but at most every rateControlFeedbackMinIntervalMs */
```

Comment:
"the value" here refers to which now?

Reply:
Is there a lot of precedent for putting typing ("Ms") into names? I would have expected `rateControlFeedbackMinInterval`. 

I think milliseconds can be inferred in the web platform thanks to [§ Use milliseconds for time measurement](https://w3ctag.github.io/design-principles/#milliseconds).

Reply:
I removed the Ms suffix.